app/modules/kit-library/assets/js/hooks/use-kit-call-to-action-updated-url.js

```diff
@@ -0,0 +1,10 @@
+export default function useKitCallToActionUpdatedUrl( url, kitId, kitTitle ) {
```

Comment:
@davseve will this `kitId` always exist?
I see in the old code it was wrapped in an if statement and only added if it exists - which indicates that it might not exist...

```
if ( model.id ) {
    url += `&utm_content=${ model.id }`;
}
```

Reply:
Yes, `kitId` and `kitTitle` will always exist
The code I wrote before had a redundant if statement.
`kitId`, `kitTitle` and the `url` are required proptypes.